feature-libs/organization/account-summary/occ/model/occ-account-summary-endpoints.model.ts

```diff
@@ -0,0 +1,31 @@
+import { OccEndpoint } from '@spartacus/core';
+
+export interface AccountSummaryOccEndpoints {
+  /**
+   * Get Account summary header details
+   *
+   * @member {string} [accountSummary]
+   */
+
+  accountSummary?: string | OccEndpoint;
```

Comment:
the name is not good, how about `getAccountSummaryHeader`? (at lease it should contain `get`). Please also check the following endpoint names as well.